src/Dfe.PlanTech.Web/Controllers/PagesController.cs

```diff
@@ -68,4 +69,9 @@ public async Task<IActionResult> NotFoundError([FromServices] IConfiguration con
 
         return View(viewModel);
     }
+
+    private async Task<INavigationLink> GetContactLinkAsync()
+    {
+        return await _getNavigationQuery.GetLinkById(_contactOptions.LinkId);
```

Comment:
```suggestion
        return await _getNavigationQuery.GetLinkById(_contactOptions.Value.LinkId);
```

And then remove the line

```
    private readonly ContactOptions _contactOptions = contactOptions.Value;
```